src/Compute/Compute/Generated/Image/ImageCreateOrUpdateMethod.cs

```diff
@@ -108,27 +125,46 @@ public override void ExecuteCmdlet()
 
         [Parameter(
             ParameterSetName = "DefaultParameter",
-            Position = 1,
+            Position = 0,
             Mandatory = true,
             ValueFromPipelineByPropertyName = true)]
         [ResourceGroupCompleter]
         public string ResourceGroupName { get; set; }
 
         [Parameter(
             ParameterSetName = "DefaultParameter",
-            Position = 2,
+            Position = 1,
             Mandatory = true,
             ValueFromPipelineByPropertyName = true)]
         [Alias("Name")]
         public string ImageName { get; set; }
 
+        [CmdletParameterBreakingChange("Image", "Image parameter will become optional when ResourceGroupName and ImageName parameters are given")]
```

Comment:
I believe the breaking change is that Image parameter will be removed for the ResourceGroupName/ImageName parameter set in an upcoming breaking change release.